### ISSUE TITLE
[Bug]: Assetbundle in Assetbundle Results in Duplicate Key in Dictionary 

### ISSUE THREAD

**Issue by SinsofSloth** [AssetRipper/AssetRipper#399]
### Are you on the latest version of AssetRipper?

Yes, I'm on the latest pre-release of AssetRipper.

### Which release are you using?

GUI Version Windows x64

### Which Unity version did this occur on?

2019.4.17f1

### Is the game Mono or IL2Cpp?

IL2Cpp

### Describe the issue.

Fails to load assets from ``romfs`` folder. The game is Rune Factory 5 v1.0.1 ENG (Nintendo Switch):
```
[Error] : System.ArgumentException: An item with the same key has already been added. Key: AssetRipper.SourceGenerated.Classes.ClassID_213.Sprite_2019_1_0_a10
   at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at AssetRipper.Core.Project.ProjectAssetContainer.AddBundleAssets(IAssetBundle bundle) in D:\a\AssetRipper\AssetRipper\AssetRipperCommon\Project\ProjectAssetContainer.cs:line 360
   at AssetRipper.Core.Project.ProjectAssetContainer..ctor(ProjectExporterBase exporter, CoreConfiguration options, VirtualSerializedFile file, IEnumerable`1 assets, IReadOnlyList`1 collections) in D:\a\AssetRipper\AssetRipper\AssetRipperCommon\Project\ProjectAssetContainer.cs:line 52
   at AssetRipper.GUI.UIAssetContainer..ctor(Ripper ripper) in D:\a\AssetRipper\AssetRipper\AssetRipperGUI\UIAssetContainer.cs:line 16
   at AssetRipper.GUI.MainWindowViewModel.<DoLoad>b__82_1(GameStructure gameStructure) in D:\a\AssetRipper\AssetRipper\AssetRipperGUI\MainWindow.ViewModel.cs:line 176
   at AssetRipper.GUI.Managers.UIImportManager.ImportFromPathInternal(Ripper ripper, String[] paths, Action`1 onComplete, Action`1 onError) in D:\a\AssetRipper\AssetRipper\AssetRipperGUI\Managers\UIImportManager.cs:line 33
```

### Relevant log output

[AssetRipper.log](https://github.com/AssetRipper/AssetRipper/files/8702952/AssetRipper.log)

**Comment by SinsofSloth** [comment]
The conflict has been determined to be the result of another assetbundle inside another assetbundle, ``c594fb0012a880d725317f210d995815.bundle``, should anyone run into the same issue. It works fine when the assetbundle is removed from this assetbundle, but I used the latest stable instead. I'm not sure if this is an intrinsic problem with the pre-release, but it's most likely not. However, the error that was shown in 1.9.9 was more useful in determining the file in question than the pre-release quickly.

**Comment by ds5678** [comment]
Thanks for reporting back

**Comment by ds5678** [comment]
Resolved in 0a5f6de9c23550f23531b288872c8e49db83f163
